src/components/SittingImg.jsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+import Sitting from "../../public/Sitting.png";
```

Comment:
To avoid writing these dots `../../` you can have it as `~/Sitting.png`. Also it's better to use SVGs instead of PNG.